### ISSUE TITLE
cap init prompt issues

### ISSUE THREAD

**Issue by nephix** [ionic-team/capacitor#1689]
Description of the problem:
When initializing capacitor in a newly created CRA, it hangs in the 2nd step when it asks for **App Package ID**.

Affected platform
- [ ] Android
- [ ] iOS
- [ ] electron
- [x] web

OS of the development machine
- [ ] Windows
- [x] macOS 10.14.5
- [ ] linux

Other information:

node version:
v12.2.0
v10.15.1
v8.15.1

npm version:
6.9.0

Steps to reproduce:
```
$ npx create-react-app test
$ cd test
$ npm i -S @capacitor/core @capacitor/cli
$ npx cap init
? App name (App) <ENTER>
? App Package ID (in Java package format, no dashes) (com.example.app) <ENTER>
```
When it asks for **App Package ID** it does not proceed by hitting enter or CTRL C.

**Comment by jcesarmobile** [comment]
I can't reproduce, are you using the terminal that comes with macOS or something else?
You have put 3 different node versions, did you test them all?

**Comment by nephix** [comment]
>  can't reproduce, are you using the terminal that comes with macOS or something else?

I tried it in the Terminal that comes with macOS as well as iterm.

> You have put 3 different node versions, did you test them all?
Yepp, same result in every version.

**Comment by jcesarmobile** [comment]
You are not entering any values, right? just using default ones
Does it work if you type anything?

**Comment by nephix** [comment]
I was able to type when it asked for **App name**.
Once it switched to **App Package ID** neither typing, `<ENTER>` nor `<CTRL + D>` worked

**Comment by christbald** [comment]
Same here ... hangs up at the very same point. Have the exact same configuration.

**Comment by Nodonisko** [comment]
Same here, same configuration

**Comment by caffeineandfoxes** [comment]
I'm also having this issue. For reference, I tested with the integrated terminal in VS Code (1.35.1) and in Terminal.app (2.7.5) with Node 8.11.3 on my iMac running macOS Sierra (10.12.6). I only have one version of Node installed on my machine, as well.

**Comment by adamflitney** [comment]
I'm also having the same issue with a fresh install of capacitor and capacitor cli. 
Tried in VSCode integrated terminal (node cmd) and also normal cmd.exe
OS: Windows 10.0.17134
Node: v10.8.0

@capacitor/core@1.0.0
@capacitor/core@1.0.0

Happened with my existing project and was able to reproduce in an empty dummy project

**Comment by jcesarmobile** [comment]
I've been able to reproduce in a different computer, will try to figure out whats going on.

As workaround you can use `npx cap init appName app.id`

**Comment by jcesarmobile** [comment]
this is a bug in[ inquirer 6.4.0](https://github.com/SBoudrias/Inquirer.js/issues/811) that was released today, will pin the dependency to 6.3.1
